isofit/utils/analytical_line.py

```diff
@@ -268,7 +268,7 @@ def run_lines(self, startstop: tuple) -> None:
             for c in range(output_state.shape[1]):
                 meas = rdn[r, c, :]
                 if self.radiance_correction is not None:
-                    meas *= self.radiance_correction
```

Comment:
This "modify in place" does not actually change the spectrum in my code versions.  The new syntax successfully performs the modification.